README.md

```diff
@@ -4,9 +4,9 @@
 Bayesian Light Source Separator (BLISS)
 ========================================
 [![](https://img.shields.io/badge/docs-master-blue.svg)](https://prob-ml.github.io/bliss/)
-![tests](https://github.com/prob-ml/bliss/workflows/tests/badge.svg)
+[![tests](https://github.com/prob-ml/bliss/workflows/tests/badge.svg)](https://github.com/prob-ml/bliss/actions/workflows/tests.yml)
 [![codecov.io](https://codecov.io/gh/prob-ml/bliss/branch/master/graphs/badge.svg?branch=master&token=Jgzv0gn3rA)](http://codecov.io/github/prob-ml/bliss?branch=master)
-![case studies](https://github.com/prob-ml/bliss/actions/workflows/case_studies.yml/badge.svg)
```

Comment:
Did we want to remove the case studies badge?

Reply:
Yes, I think. No point in running the "case studies" job nightly any more. It's the same as calling `pytest --gpu`, which we'll now do on push.